src/social-network-provider/twitter.com/ui/inject.tsx

```diff
@@ -6,6 +6,9 @@ import { renderInShadowRoot } from '../../../utils/jss/renderInShadowRoot'
 import { SocialNetworkUIInjections } from '../../../social-network/ui'
 import { nop } from '../../../utils/utils'
 
+/**
+ * using a closed shadow root can prevent access from other script.
+ */
 const newMOW = (i: LiveSelector<HTMLElement, true>) =>
     new MOW(i).setDomProxyOption({ afterShadowRootInit: { mode: 'closed' } }).startWatch()
```

Comment:
Don't forget `beforeShadowRootInit` if you use `.before` somewhere

Reply:
comment has been update.

Reply:
??? So just add a `beforeShadowRootInit`, it doesn't cost anything